python/marvin/tools/rss.py

```diff
@@ -1,102 +1,116 @@
 #!/usr/bin/env python
-# encoding: utf-8
+# -*- coding: utf-8 -*-
 #
-# rss.py
+# @Author: Brian Cherinka, José Sánchez-Gallego, and Brett Andrews
+# @Date: 2016-04-11
+# @Filename: rss.py
+# @License: BSD 3-clause (http://www.opensource.org/licenses/BSD-3-Clause)
 #
-# Licensed under a 3-clause BSD license.
-#
-# Revision history:
-#     11 Apr 2016 J. Sánchez-Gallego
-#       Initial version
+# @Last modified by: José Sánchez-Gallego (gallegoj@uw.edu)
+# @Last modified time: 2018-07-24 12:16:14
 
 
 from __future__ import division, print_function
 
+import os
 import warnings
 
-import numpy as np
+import astropy.io.ascii
+import astropy.table
+import astropy.units
+import astropy.wcs
+import numpy
 from astropy.io import fits
 
 import marvin
 from marvin.core.exceptions import MarvinError, MarvinUserWarning
-from marvin.tools.quantities.spectrum import Spectrum
+from marvin.utils.datamodel.drp import datamodel_rss
+from marvin.utils.datamodel.drp.base import Spectrum as SpectrumDataModel
 
 from .core import MarvinToolsClass
+from .cube import Cube
+from .mixins import GetApertureMixIn, NSAMixIn
+from .quantities.spectrum import Spectrum
+
+
+class RSS(MarvinToolsClass, NSAMixIn, GetApertureMixIn, list):
```

Comment:
why is the `GetApertureMixIn` added in here?  This doesn't really make sense to me for RSS files.  These are single fiber spectra presented in a linear stack in order of exposure + MJD + slithead fiber id rather than in any kind of spatial ordering.  A circular aperture seems irrelevant.   

What I could see, and maybe we want to do this, is add into the `Spaxel` class an attribute or method that grabs all the RSS fibers that associated with that spaxel.  I think that was a Renbin original.  Then a user could define an aperture in cube space, grab the spaxels, and then grab the original RSS fibers for all the spaxels in the aperture.

Reply:
Good catch. I simply copied it from the Cube and forgot to remove the mixin. Fixed.